cmd/config.go

```diff
@@ -90,6 +90,14 @@ func ConfigSet(appID string, configVars []string) error {
 		configMap["SSH_KEY"] = base64.StdEncoding.EncodeToString([]byte(sshKey))
 	}
 
+	// NOTE(bacongobbler): check if the user is using the old way to set healthchecks. If so,
+	// send them a deprecation notice.
+	if _, ok := configMap["HEALTHCHECK_URL"]; ok {
```

Comment:
Yeah, then people would be alerted if they were adjusting any part of the healthchecks instead of just url.

It's optional tho, I'm not going to push it as a required change